packages/core/src/tracing/instrumentation/messaging/kafkaJs.js

```diff
@@ -20,6 +22,14 @@ let headerFormat = constants.kafkaHeaderFormatDefault;
 
 let isActive = false;
 
+const allInstanaHeaders = [
```

Comment:
Would it make sense to have this array in `constants.js` for convenience? Or would it be just overhead?
I saw them repeated a few times in the tests, which is why I was wondering.
Apart from this comment, "packing" Instana headers into an array is a simple great idea. It makes it pretty visible that the Instana headers must be removed from the original headers after the consumer is instrumented.

Reply:
Yes, extracting them to constants makes sense. I'll take care of that.

> Apart from this comment, "packing" Instana headers into an array is a simple great idea.

Haha, I agree it is a great idea, it is also _your_idea, my friend. Your original rdkafka instrumentation did the same (I just moved the array out of the `remove...` function there.

Reply:
:O I actually saw that later in the PR :D